pkg/reconciler/ingress/ingress.go

```diff
@@ -493,13 +491,11 @@ func (r *BaseIngressReconciler) ensureFinalizer(ra ReconcilerAccessor, ia v1alph
 func (r *BaseIngressReconciler) reconcileGateway(ctx context.Context, ia v1alpha1.IngressAccessor, gw config.Gateway, desired []v1alpha3.Server) error {
 	// TODO(zhiminx): Need to handle the scenario when deleting ClusterIngress. In this scenario,
 	// the Gateway servers of the ClusterIngress need also be removed from Gateway.
-	logger := logging.FromContext(ctx)
 	gateway, err := r.GatewayLister.Gateways(gw.Namespace).Get(gw.Name)
 	if err != nil {
-		// Not like VirtualService, A default gateway needs to be existed.
+		// Not like VirtualService, A default gateway needs to be existent.
```

Comment:
```suggestion
		// Unlike VirtualService, a default gateway needs to be existent.
```
Since we're fixing comments :-)

Reply:
Done